include/ddc/kernels/splines/bsplines_non_uniform.hpp

```diff
@@ -117,7 +117,7 @@ class NonUniformBSplines : detail::NonUniformBSplinesBase
          *
          * @param breaks The std::initializer_list of the coordinates of break points.
          */
-        explicit Impl(std::initializer_list<ddc::Coordinate<CDim>> breaks)
+        Impl(std::initializer_list<ddc::Coordinate<CDim>> breaks)
```

Comment:
I thought we need an explicit here. Could you please comment why not needed?

Reply:
This is also how the check works https://clang.llvm.org/extra/clang-tidy/checks/google/explicit-constructor.html. They advise implicit constructors with initializer lists. I think it is be able to write
```cpp
struct A
{
    A(std::initializer_list<int> l);
};

void foo(A const& a);

// using foo
foo({1, 2, 3});
```